index.html

```diff
@@ -7,6 +7,8 @@
   <script src='https://www.w3.org/Tools/respec/respec-w3c-common' async class='remove'></script>
   <script class='remove'>
   var respecConfig = {
+    wg: "Web Platform Incubator Community Group",

```

Comment:
We did formally agree to adopt Reporting and NEL into webperf WG. We should do the move and update the WG reference.

@plehegar @siusin @yoavweiss can you help coordinate the transfer?

Reply:
Updated the references

Reply:
Happy to help on the WICG side. 

@dcreager - will you be able to send out an [intent to migrate](http://wicg.github.io/admin/intent-to-migrate.html) to https://discourse.wicg.io?

@siusin /@plehegar - what's the procedure from the W3C side of things? Do I just transfer the repo to the W3C org?

Reply:
@sideshowbarker has the admin access to both orgs (W3C/WICG), please give him a ping when the intent to migrate has been sent out.

Reply:
Just filed an [intent to migrate](https://discourse.wicg.io/t/intent-to-migrate-reporting-and-network-error-logging/2864?u=dcreager).  Please take a look and let me know if I missed anything!

Reply:
Also opened wicg/reporting#100 to track a similar move for the Reporting spec.

Reply:
Pinged @sideshowbarker on IRC to try to get him to move the repos over

Reply:
Repos have been moved over